File: data_sample_16/translate/train.java-cs.txt.java
public static DVConstraint createExplicitListConstraint(String[] explicitListValues) {return new DVConstraint(null, explicitListValues);}
public String toString(){StringBuilder buffer = new StringBuilder();buffer.append("[SCL]\n");buffer.append("    .numerator            = ").append("0x").append(HexDump.toHex(  getNumerator ())).append(" (").append( getNumerator() ).append(" )");buffer.append(System.getProperty("line.separator"));buffer.append("    .denominator          = ").append("0x").append(HexDump.toHex(  getDenominator ())).append(" (").append( getDenominator() ).append(" )");buffer.append(System.getProperty("line.separator"));buffer.append("[/SCL]\n");return buffer.toString();}
public UDFFinder getUDFFinder(){return _uBook.getUDFFinder();}
public void reset(int point) {this.point = point;ends.next = 0;starts.next = 0;}
public void write(String str, int offset, int count) {write(str.substring(offset, offset + count).toCharArray());}
public void addRecords(RecordStream rs) {while (true) {if (!readARecord(rs)) {break;}}}
public ListStackSetsResult listStackSets(ListStackSetsRequest request) {request = beforeClientExecution(request);return executeListStackSets(request);}
public void setCollector(Collector collector) {this.collector = collector;}
public String toString() {if (noBreak == null && postBreak == null && preBreak != null&& preBreak.equals("-")) {return "-";}StringBuilder res = new StringBuilder("{");res.append(preBreak);res.append("}{");res.append(postBreak);res.append("}{");res.append(noBreak);res.append('}');return res.toString();}
public DescribeRegionsResult describeRegions() {return describeRegions(new DescribeRegionsRequest());}
public VCenterRecord clone() {return copy();}
public AutomatonQuery(final Term term, Automaton automaton) {this(term, automaton, Operations.DEFAULT_MAX_DETERMINIZED_STATES);}
public String toString() {return getClass().getSimpleName() + "[" + getFile().getPath() + "]";}
public void fromRaw(int[] ints, int p) {w1 = ints[p];w2 = ints[p + 1];w3 = ints[p + 2];w4 = ints[p + 3];w5 = ints[p + 4];}
public void remove(int index) {checkIndex(index);_mergedRegions.remove(index);}
public int size() {return mSize;}

File: data_sample_16/translate/train.java-cs.txt.cs
public static DVConstraint CreateExplicitListConstraint(String[] explicitListValues){return new DVConstraint(null, explicitListValues);}
public override String ToString(){StringBuilder buffer = new StringBuilder();buffer.Append("[SCL]\n");buffer.Append("    .numerator            = ").Append("0x").Append(HexDump.ToHex(Numerator)).Append(" (").Append(Numerator).Append(" )");buffer.Append(Environment.NewLine);buffer.Append("    .denominator          = ").Append("0x").Append(HexDump.ToHex(Denominator)).Append(" (").Append(Denominator).Append(" )");buffer.Append(Environment.NewLine);buffer.Append("[/SCL]\n");return buffer.ToString();}
public UDFFinder GetUDFFinder(){return _uBook.GetUDFFinder();}
public void Reset(int point){this.point = point;ends.count = 0;starts.count = 0;}
public override void write(string str, int offset, int count){write(Sharpen.StringHelper.Substring(str, offset, offset + count).ToCharArray());}
public void AddRecords(RecordStream rs){while (true){if (!ReadARecord(rs)){break;}}}
public virtual ListStackSetsResponse ListStackSets(ListStackSetsRequest request){var options = new InvokeOptions();options.RequestMarshaller = ListStackSetsRequestMarshaller.Instance;options.ResponseUnmarshaller = ListStackSetsResponseUnmarshaller.Instance;return Invoke<ListStackSetsResponse>(request, options);}
public virtual void SetCollector(ICollector collector){this.collector = collector;}
public override string ToString(){if (NoBreak == null && PostBreak == null && PreBreak != null && PreBreak.Equals("-", StringComparison.Ordinal)){return "-";}StringBuilder res = new StringBuilder("{");res.Append(PreBreak);res.Append("}{");res.Append(PostBreak);res.Append("}{");res.Append(NoBreak);res.Append('}');return res.ToString();}
public virtual DescribeRegionsResponse DescribeRegions(){return DescribeRegions(new DescribeRegionsRequest());}
public override Object Clone(){VCenterRecord rec = new VCenterRecord();rec.field_1_vcenter = field_1_vcenter;return rec;}
public AutomatonQuery(Term term, Automaton automaton): base(term.Field){this.m_term = term;this.m_automaton = automaton;this.m_compiled = new CompiledAutomaton(automaton);}
public override string ToString(){return GetType().Name + "[" + GetFile().GetPath() + "]";}
public virtual void FromRaw(int[] ints, int p){w1 = ints[p];w2 = ints[p + 1];w3 = ints[p + 2];w4 = ints[p + 3];w5 = ints[p + 4];}
public void Remove(int index){CheckIndex(index);_mergedRegions.RemoveAt(index);}
public virtual int size(){return mSize;}
